packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
+/* stylelint-disable selector-class-pattern  */
+.card {
```

Comment:
Just curious, are we going to start making most style's themeable right out of the gate, or add in theme-ability as the need comes? We can always add more customizability, but it's harder to take it away